src/frontend/components/PubPage.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { kernel, useFriendsList } from '../db'
+import VibeList from './Vibe.jsx'
+
+export default function Pubs () {
+  const peers = useFriendsList()
+  const icons = {
+    0: '♀️',
+    1: '♂️',
+    2: '⚧️'
+  }
+  const sendVibe = (peer) => {
+    console.log('pk0',peer.pk)
+    kernel.sendVibe(peer.pk)
+      .then(chatId => {
+        console.log(chatId)
+      })
+      .catch(err => {
+        console.log(err)
+      })
+  }
+  return (
+    <>
+      <div className='hero'>
+        <h1 className='bar'>Pub name is <strong>"HardRock"</strong></h1>
+        <h3 className='membersList'>Here is memmbers list which is at this bar now</h3>
+        <VibeList />
+        <div className='friendsList friendsListWrap'>
+          {peers.map(friend => (
+            <div className='friendsListWrap2' key={friend.pk}>
+              <div className='column friendsList-2'>
+                <div className='column friendsIcon'>
+                  <span className='icon-3'>{friend.picture}</span>
+                </div>
+                <div className='column'>
+                  <h1>{friend.name}</h1>
+                  <span className='icon-2'>{icons[friend.sex]}</span>
+                  <h3 className='smalleTagline'>{friend.tagline}</h3>
```

Comment:
don't use `h3` for small text. `h3` means "Header3" which makes text larger than normal.
instead use:
```jsx
<div><small>{friend.tagline}</small></div>
```
if you still need a css-class then fix case and typo: `small-tagline`